public/static/js/mci_module.js

```diff
@@ -302,8 +302,6 @@ var mciModule = angular.module('MCI', [
   };
 
   return $time;
-}]).config(['$compileProvider', function ($compileProvider) {
```

Comment:
Did you mean to remove this code?

Reply:
Seems to not be doing anything -- is it needed for something? I removed it since it seemed to be hanging from another ticket where it was commented out but never removed.

Reply:
As long as you are sure that there are no bad side effects from removing it, go for it.  In general I would limit changes in a PR to issues that actually relate to the ticket being implemented.